src/main/java/uk/gov/companieshouse/web/pps/controller/pps/ConfirmationController.java

```diff
@@ -63,28 +80,46 @@ public String getConfirmation(@PathVariable String companyNumber,
             return ERROR_VIEW;
         }
 
-
```

Comment:
Line 59 - Can you remove the unused parameter 'reference'?

Reply:
Removed as suggested